slices/liblzma5.yaml

```diff
@@ -1,7 +1,14 @@
 package: liblzma5
+essential:
+  - liblzma5_copyright
+
```

Comment:
nit: it would be nice to have a blank line separating `package` and `essential`.

Reply:
done